src/Arriba/Arriba.Server/Program.cs

```diff
@@ -64,158 +45,6 @@ private static IHostBuilder CreateHostBuilder(string[] args)
                     webBuilder.UseStartup<Startup>();
                 });
         }
-
-        private class Startup
```

Comment:
This commit appears to mix a refactoring (code move/rename) with edits, which makes the edits much harder to detect and observe. I would recommend splitting this commit into at least two, but possibly more commits.